### PR TITLE
docs(AR): new folder structure Arabic

### PR DESCRIPTION
**Pull request by Sboonny**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- Changed the file to `.mjx`
- Fixed heading structure.
- Removed ملف and مجلد from the heading because some were misused and describing the wrong state of the folder/file.

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: www/src/pages/ar/folder-structure.mdx [hunk 1]
```diff
@@ -6,227 +6,267 @@ lang: ar
...
 
 ملف `trpc.ts` يُسخدم كـ entrypoint لـ tRPC، إقرأ المزيد هنا [tRPC usage](usage/trpc#-utilstrpcts)
 
 <sub>(في حالة tRPC)</sub>
 
-### ملف `.env`
+</div>
+<div>
+
+### `.env`
```
Comment (Sboonny) [inline www/src/pages/ar/folder-structure.mdx:215]: Random question, should this be heading two? `##`

I think the heading is used here to indicate where the file is, so if it's heading 2, then it's in the root section of the app.

File: www/src/pages/ar/folder-structure.mdx [hunk 2]
```diff
@@ -6,227 +6,267 @@ lang: ar
...
 
 ملف `.env` يُستخدم لتخزين `environment variables`، إقرأ المزيد [Environment Variables](usage/env-variables)
 
-### ملف `.env.example`
+</div>
+<div>
+
+### `.env.example`
```
Comment (Sboonny) [inline www/src/pages/ar/folder-structure.mdx:222]: Same, and the sections below it

**Review by Sboonny — COMMENTED** [review]
Random question about the original structure of the page

**Comment by juliusmarminge** [comment]
> Random question about the original structure of the page

I think we did this to make the sizing more appropriate cause there were some overflow when we followed the "actual proper heading numbers" so to say...

**Review by juliusmarminge — CHANGES_REQUESTED** [review]
I think all the `sub` ones should go since now you pick your selections in the Form component

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**